templates/Package/API/QueryBuilder.php.twig

```diff
@@ -42,9 +40,13 @@ class {{ entityName.singular }}QueryBuilder
      */
      public function count(): int
      {
-         $response = $this->client->get("{{ entityName.plural }}/query/count", [
-             'search' => json_encode( $this->toArray() )
-         ]);
+        if (strlen($this->__toString()) >= 1800) {
```

Comment:
Is 1800 a hard-limit set by API? If so, worth moving it to a constant.

Reply:
No. Standard URI limit is 2000, set it to 1800 as a slight buffer in case of any edge cases (e.g., using the /count path instead of base /query path).

Reply:
There's no such thing as standard URI limit. At least in RFC. 2048 is the limit in old good IE. Anyway, if you are sure 1800 covers all the possible edge cases, it is fine.